pages/education.py

```diff
@@ -319,44 +311,51 @@ def education_narrative(data, country):
         end_year = spending.year.max()
         start_value = spending[spending.year == start_year].real_expenditure.values[0]
         end_value = spending[spending.year == end_year].real_expenditure.values[0]
+        spending_growth_rate = ((end_value - start_value) / start_value)
+        trend = 'increased' if end_value > start_value else 'decreased'
+        text = f"Between {start_year} and {end_year} after adjusting for inflation, total public spending on education in {country} has {trend} from ${millify(start_value)} to ${millify(end_value)}, reflecting a growth rate of {spending_growth_rate:.0%}. "
+
+        spending['real_central_expenditure'] = spending.real_expenditure / spending.expenditure * spending.central_expenditure
```

Comment:
Why are we showing real_central_expenditure for the narrative only? 
We are showing the raw central expenditure in the figure. The narrative seems inconsistent with the figure.

![image](https://github.com/user-attachments/assets/b2ec603b-dfc0-4b7c-973c-f30b5d95838c)

Reply:
I had the same thought when I worked on it – the graph is kept consistent with what's in the overview (expenditure in bars broken down by central vs regional, while real exp only shows total in line. The downside with using inflation adjusted only is that we might lose the latest year due the lag of CPI data. One solution could be to add a radio button to allow toggle between inflation-adjusted vs. current expenditure, like what we have in the overview - across space page for total vs. per-capital exp.

Reply:
I see. I think radio button sounds like a good solution. (maybe in a separate PR?) 
I kind of see the narrative as the complement to the figure, so it would be good to display the same number in both narrative and in the figure.